config.go

```diff
@@ -141,6 +141,15 @@ func getConfig() *types.Configuration {
 	v.SetDefault("Kubeless.Port", 8080)
 	v.SetDefault("Kubeless.Kubeconfig", "")
 	v.SetDefault("Kubeless.MinimumPriority", "")
+
+	v.SetDefault("Openfaas.GatewayNamespace", "openfaas")
+	v.SetDefault("Openfaas.GatewayAddress", "gateway")
+	v.SetDefault("Openfaas.FunctionName", "")
+	v.SetDefault("Openfaas.FunctionNamespace", "openfaas-fn")
+	v.SetDefault("Openfaas.Port", 8080)
```

Comment:
```suggestion
	v.SetDefault("Openfaas.GatewayPort", 8080)
```

It makes more sense as it's related to Gateway

Reply:
Yep, you are right 👍

Reply:
done.